packages/d3fc-webgl/src/series/glLine.js

```diff
@@ -140,6 +131,32 @@ export default () => {
         return draw;
     };
 
+    draw.defined = (...args) => {
+        const builder = program.buffers().attribute(definedAttrib);
+
+        const definedArray = new Float32Array(args[0].length * 4);
+        const values = args[0];
+
+        for (let i = 0; i < values.length; i += 1) {
```

Comment:
I've noticed this could also be done using `forEach`, is there a preference one way or another for using a `for` loop here or `values.forEach`?

_For reference:_
```javascript
values.forEach((value, i) => {
  const previousValue = i === 0 ? value : values[i - 1];
  const nextValue = i === (values.length - 1) ? value : values[i + 1];
  const bufferIndex = i * 4;

  definedArray[bufferIndex] = value ? previousValue : value;
  definedArray[bufferIndex + 1] = value ? previousValue : value;
  definedArray[bufferIndex + 2] = value ? nextValue : value;
  definedArray[bufferIndex + 3] = value ? nextValue : value;
});
```